.golangci.yml

```diff
@@ -37,6 +40,8 @@ issues:
         - unparam                       # Tests might have unused function parameters.
         - lll
         - dupl
+        - misspell
```

Comment:
I think it is fine to enable `misspell` in test file if doesn't require much changes

Reply:
It does require changes. Enabling these linters for tests would need a lot of extra work. I'll create a separate issue for the same?